corehq/apps/app_manager/tests/data/suite/sync_request.xml

```diff
@@ -0,0 +1,19 @@
+<partial>
+  <sync-request>
+    <post url="https://www.example.com/a/test_domain/phone/claim-case/">
+      <data key="case_id" ref="session/datum/case[@id]"/>
+      <data key="case_type" ref="session/datum/case/type"/>
```

Comment:
@phillipm @ctsims I'm making a bunch of assumptions re the sync-request node that the device expects in suite.xml. Can you give me some examples of what `ref` needs to look like? From what I understand this need to point to case ID and case type of the search result that the user has chosen.